packages/avocode-email-tagsinput/test/e2e/unique.spec.js

```diff
@@ -0,0 +1,57 @@
+import puppeteer from 'puppeteer'
+import { expect } from 'chai'
+import {
+  URL,
+  tagsInputFieldSelector,
+  addEmailTag,
+  assertEmailTagCount,
+} from './test-utils'
+
+describe('Unique Email Tags Input', () => {
+  let browser
+  let page
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch(browserOptions)
+  })
+
+  afterAll(async () => {
+    await browser.close()
+  })
+
+  beforeEach(async () => {
+    page = await browser.newPage()
+    await page.goto(`${URL}/unique`)
+  })
+
+  afterEach(async () => {
+    await page.close()
+  })
+
+  it('should render editor', async () => {
+    const editor = await page.waitForSelector(tagsInputFieldSelector)
+    expect(editor).to.exist
+  })
+
+  describe('user typing different email edge cases', () => {
+    const uniqueEmailExamples = [
+      "single'123@quote.aa",
```

Comment:
When using testing emails, please use some generic ones, these look spammy and I'm not sure about them. Why not `foo@bar.com` or something like that?